planning/behavior_path_planner/src/utils/lane_change/utils.cpp

```diff
@@ -1084,7 +1083,9 @@ ExtendedPredictedObject transform(
       if (t < prepare_duration && obj_vel_norm < velocity_threshold) {
         continue;
       }
-      const auto obj_pose = object_recognition_utils::calcInterpolatedPose(path, t);
+      const auto obj_pose = obj_vel_norm > velocity_threshold
```

Comment:
For this threshold, we can use `lane_change_parameters.stop_velocity_threshold`.